client/python/cryoet_data_portal/src/cryoet_data_portal/_file_tools.py

```diff
@@ -18,35 +21,42 @@ def parse_s3_url(url: str) -> (str, str):
     return parsed.netloc, parsed.path
 
 
-def download_https(url: str, dest_path: Optional[str] = None, with_progress: bool = True):
+def download_https(
+    url: str,
+    dest_path: Optional[str] = None,
+    with_progress: bool = True,
+):
     dest_path = get_destination_path(url, dest_path)
     fetch_request = requests.get(url, stream=True)
     total_size = int(fetch_request.headers["content-length"])
     block_size = 1024 * 512
-    print(f"Downloading {url} to {dest_path}")
+    logger.info("Downloading %s to %s", url, dest_path)
```

Comment:
formatted strings in log statements upset the linters because technically python still has to convert the objects to strings even if the log is not run